tests/async/test_depends.py

```diff
@@ -469,3 +470,17 @@ async def func(a=Depends(dep)):
         return a
 
     assert await func() == "a"
+
+
+@pytest.mark.anyio
+async def test_asynccontextmanager():
+    async def dep(a: str = "a"):
```

Comment:
Please, remove default value here and in sync test to make the test much representable

Reply:
Done